kpops/component_handlers/utils/exception.py

```diff
@@ -1,20 +1,20 @@
 import logging
 
-import requests
+import httpx
 
 log = logging.getLogger("RequestException")
```

Comment:
this also needs to be renamed to match the class name

Reply:
good point!